LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -2063,63 +2027,64 @@
         });
 
         $.ajax({
-                type: 'GET',
-                url: '../api/workflow/' + workflowID,
-                success: function(res) {
-                    var minY = 80;
-                    var maxY = 80;
-                    for (let i in res) {
-                        steps[res[i].stepID] = res[i];
-                        posY = parseFloat(res[i].posY);
-                        if (posY < minY) {
-                            posY = minY;
-                        }
+            type: 'GET',
+            url: '../api/workflow/' + workflowID,
+            success: function(res) {
+                var minY = 80;
+                var maxY = 80;
+                for (let i in res) {
+                    steps[res[i].stepID] = res[i];
+                    posY = parseFloat(res[i].posY);
+                    if (posY < minY) {
+                        posY = minY;
+                    }
 
-                        let emailNotificationIcon = '';
-                        if (typeof res[i].stepData == 'string') {
-                            let stepParse = JSON.parse(res[i].stepData);
-                            if (stepParse.AutomatedEmailReminders?.AutomateEmailGroup?.toLowerCase() ===
-                                'true') {
-                                let dayCount = stepParse.AutomatedEmailReminders.DaysSelected;
-                                let dayText = ((dayCount > 1) ? 'Days' : 'Day')
-                                emailNotificationIcon = `<img src="../dynicons/?img=appointment.svg&w=18" style="margin-bottom: -3px;" alt="Email reminders will be sent after ${dayCount} ${dayText} of inactivity" />`
-                            }
+                    let emailNotificationIcon = '';
+                    if (typeof res[i].stepData == 'string') {
+                        let stepParse = JSON.parse(res[i].stepData);
```

Comment:
Just because it's a string doesn't mean it's json.  Not sure how you use it below, but you might want to do another check here.

Reply:
There's a checkForJSON method in enhance/LEAF-3732/file_mngr_add_info